flow-typed/custom/superagent.js

```diff
@@ -0,0 +1,307 @@
+import * as fs from "fs";
+import * as http from "http";
+import * as stream from "stream";
+import * as cookiejar from "cookiejar";
+
+declare type superagent$CallbackHandler = (
+    err: any,
+    res: superagent$Response,
+) => ?boolean;
```

Comment:
I changed this. The typescript defs have this returning void, but I know from the `retry` method that it can return a maybe boolean in order to specify the `shouldRetry` callback.